docs/tutorials/schrödinger_equation.rst

```diff
@@ -0,0 +1,276 @@
+
+    @alias.register_function(lib="numpy", path="runge_kutta")
+    def _(state, N):
+        probabilities = []
+        for n in range(N):
+            probabilities.append(state_probabilities(state))
+            state+= runge_kutta_step(Z, X, n, state)
+        return probabilities
+
+
+
+This custom function of ``runge_kutta`` is registered using the decorator ``@alias.register_function`` under 
+NumPy.
+
+In the case of JAX, we want to use ``jax.lax.scan`` function for instead of Python for efficient loops.
+
+.. jupyter-execute::
+
+    @alias.register_function(lib="jax", path="runge_kutta")
+    def _(state, N):
+        def runge_kutta_step_scan(carry, probabilities):
+            n, state = carry
+            probabilities = state_probabilities(state)
+            state+= runge_kutta_step(Z, X, n, state)
+            return (n + 1, state), probabilities
+        _, probabilities = jax.lax.scan(runge_kutta_step_scan, (0, state), jnp.zeros((N,2)))
+        return probabilities
+
+6. Solve using the custom function
+----------------------------------
+
+We have just completed writing the generic code to solve the Schrödinger equation.
+We will now demonstrate two cases of solving the equation using NumPy and JAX as inputs.
+
+The Numpy case is here:
+
+.. jupyter-execute::
+
+    Z = np.array([[1,0],[0,-1]])
+    X = np.array([[0,1],[1,0]])
+    init_state = np.array([1. + 0j,0. + 0j])
+
+    probabilities = unp.array(unp.runge_kutta(init_state, N))
+
+    T = np.linspace(0,(N-1) * dt,N)
+    plt.plot(T, probabilities, label = ["0", "1"])
+    plt.xlabel('T')
+    plt.ylabel('Probability')
+    plt.legend()
+    plt.show()
+
+.. jupyter-execute::
+
+    %timeit unp.array(unp.runge_kutta(init_state, N))
+
+
+Second case is JAX:
+
+.. jupyter-execute::
+
+    Z = jnp.array([[1,0],[0,-1]])
+    X = jnp.array([[0,1],[1,0]])
+    init_state = jnp.array([1. + 0j,0. + 0j])
+    probabilities = unp.array(unp.runge_kutta(init_state, N))
+
+    T = np.linspace(0,(N-1) * dt,N)
+
+    plt.plot(T, probabilities, label=["0", "1"])
+    plt.xlabel('T')
+    plt.ylabel('Probability')
+    plt.legend()
+    plt.show()
+
+We see if we can actually jit.
+
+.. jupyter-execute::
+
+    from functools import partial
+
+    @partial(jax.jit, static_argnums=(1,))
+    def solve_with_RungeKutta_jit(init_state, N):
+        return unp.array(unp.runge_kutta(init_state, N))
+
+.. jupyter-execute::
+
+    %timeit solve_with_RungeKutta_jit(init_state, N)
+
+By following these steps, we've learned how to leverage Arraylias to 
+write versatile numerical code that can efficiently switch between different arrays.
+We've also explored how to accelerate our code's execution using JAX's JIT compilation.
+By understanding the strengths of different numerical libraries and harnessing their capabilities through Arraylias, we can create high-performance code for various computational tasks. 
+Apply these concepts to your own projects to unlock their full potential.
```

Comment:
I don't think this text is necessary - again this is describing general benefits of using arraylias, but this tutorial should just describe the concrete steps necessary to carry out the task at hand.

Reply:
I updated at https://github.com/Qiskit-Extensions/arraylias/pull/25/commits/328ab5b771dca327efab87a6d5d6a871184d06de.